tests/python/pytorch/graphbolt/test_feature_store.py

```diff
@@ -1,13 +1,32 @@
+import os
+import tempfile
+
+import numpy as np
 import pytest
 import torch
 from dgl import graphbolt as gb
 
 
-def test_in_memory_feature_store():
+def to_on_disk_tensor(name, t):
+    with tempfile.TemporaryDirectory() as test_dir:
```

Comment:
I'm not sure if it works as file will be deleted out of `with` but returned value `t` is still referencing to disk tensor? file deletion will fail?

Reply:
It is weird because it still work... Maybe it is because the content on the file is still there. I will change the code to avoid this issue.